### PR TITLE
added support for configuration file

### PR DESCRIPTION
**Pull request by Abhijay**
(Just putting in the PR so I know no more changes need to be made. In terms of merging, this is a lower priority than some of the other changes we're working on.)

- modified FHIRExporter constructor
- made export profile properties based on config
- bump package number

### CODE REVIEW

File: lib/export.js [hunk 1]
```diff
@@ -22,13 +22,26 @@ const allowedConversions = {
...
-  const exporter = new FHIRExporter(specifications);
+const defaultConfig = {
+  "projectName": "Example Project",
+  "projectShorthand": "EXAMPLE",
+  "projectURL": "http://example.com",
+  "publisher": "Example Publisher",
+  "contact": [{
+    "telecom": [{
+      "system": "url",
+      "value": "http://standardhealthrecord.org"
```
Comment (cmoesel) [inline lib/export.js:33]: This probably shouldn't be the default right?  Instead, "http://example.com".

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "shr-fhir-export",
-  "version": "5.0.0-beta.5",
+  "version": "5.0.0-beta.6",
```
Comment (cmoesel) [inline package.json:3]: When this is rebased, we'll need to bump the version again.

**Review by cmoesel — CHANGES_REQUESTED** [review]
I don't think this quite goes far enough, I suspect Mark wanted more, but you should discuss w/ him.

For example, the FHIR IG that gets created by the IG publisher will still create a lot of SHR-based content.  That's because its config files have SHR stuff hard-coded in:
* lib/ig_files/shr.xml
* lib/ig_files/shr.json

In addition, some of the static IG content references SHR or standardhealth:
* lib/ig_files/pages/_includes/header.html
* lib/ig_files/pages/index.html
* lib/ig_files/pages/codesystems.html
* lib/ig_files/pages/extensions.html
* lib/ig_files/pages/profiles.html
* lib/ig_files/pages/valuesets.html

And some of the other generated content as well:
* lib/ig.js
* lib/common.js (e.g., fhirURL function) 
* lib/codeSystems.js
* lib/extensions.js
* lib/valueSets.js

One way to test this (although I haven't tried), might be to create some dummy definitions (elements and valuesets) that *don't* have SHR namespace or references, and a config file without SHR mentioned -- then run the tool and search the output for "SHR",  "Standard Health Record", and "standardhealth".  Once you get it out of that stuff, run the IG publisher and search what it generates.

**Comment by Abhijay** [comment]
I have since overhauled the config system in a different branch (one that is properly rebased as well). As a result, this PR should be closed and I will be making a new one from that branch.

**Pull request closed without merging**